src/plonkish/compiler/cell_manager.rs

```diff
@@ -262,20 +264,21 @@ impl CellManager for MaxWidthCellManager {
 
         let mut forward_signal_column: usize = 0;
         let mut forward_signal_row: usize = 0;
+        let mut forward_columns: Vec<Column> = Vec::new();
 
         for forward_signal in unit.forward_signals.iter() {
-            let column = if placement.columns.len() <= forward_signal_column {
+            let column = if placement.forward.len() <= forward_signal_column {
```

Comment:
@atsupontio you are comparing signals with columns here, the left side should be forward_columns.len()

Reply:
advice_columns.len()